stdlib/public/SwiftShims/swift/shims/Visibility.h

```diff
@@ -203,18 +203,18 @@
                  SWIFT_ATTRIBUTE_FOR_EXPORTS,                         \
                  SWIFT_ATTRIBUTE_FOR_IMPORTS)
 
-// SWIFT_EXPORT_FROM(LIBRARY) declares something to be a C-linkage
-// entity exported by the given library.
-//
-// SWIFT_RUNTIME_EXPORT is just SWIFT_EXPORT_FROM(swiftCore).
-//
-// TODO: use this in shims headers in overlays.
 #if defined(__cplusplus)
-#define SWIFT_EXPORT_FROM(LIBRARY) extern "C" SWIFT_EXPORT_FROM_ATTRIBUTE(LIBRARY)
+#define SWIFT_BEGIN_DECLS extern "C" {
+#define SWIFT_END_DECLS }
 #else
-#define SWIFT_EXPORT_FROM(LIBRARY) SWIFT_EXPORT_FROM_ATTRIBUTE(LIBRARY)
```

Comment:
`SWIFT_EXPORT_FROM()` is used in several other places, so you cannot drop this definition.

Reply:
But `SWIFT_EXPORT_FROM_ATTRIBUTE` is not used outside of this file, so you could just rename that to `SWIFT_EXPORT_FROM`